src/ManageCourses.Api/Controllers/DataController.cs

```diff
@@ -35,6 +36,7 @@ public OrganisationCourses ExportByOrganisation(string ucasCode)
         /// Imports the data.
         /// </summary>
         [Authorize(AuthenticationSchemes = BearerTokenApiKeyDefaults.AuthenticationScheme)]
+        [ExemptFromAcceptTerms]
```

Comment:
Can we make it so the attribute works on the entire controller

minor but there's rather a lot of these

Reply:
urgh! but 👍 thanks

Reply:
you know what, this requires an extra 30 lines of mocking/reflection code to test so feel free to go and "card it up" as the kids say